contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.2.0--2.3.0.sql

```diff
@@ -4359,10 +8015,26 @@ AS $BODY$ BEGIN
 END; $BODY$
 LANGUAGE plpgsql;
 
+-- And reset babelfishpg_tsql.restored_server_collation_name and babelfishpg_tsql.restored_default_locale GUC
+do
+language plpgsql
+$$
+    declare
+        query text;
+    begin
+        query := pg_catalog.format('alter database %s reset babelfishpg_tsql.restored_server_collation_name', CURRENT_DATABASE());
+        execute query;
+        query := pg_catalog.format('alter database %s reset babelfishpg_tsql.restored_default_locale', CURRENT_DATABASE());
```

Comment:
Let's avoid setting/resetting restored GUCs other than `2.0.0--2.1.0` upgrade scripts.

Reply:
will fix in new rev